app/app/(root)/profile/setting/error.tsx

```diff
@@ -0,0 +1,5 @@
+"use client";
+
+export default function Error() {
+  return <div>Something went wrong</div>;
+}
```

Comment:
maybe add a reset button for user to recover from error ?
```suggestion
'use client'
 
export default function Error({
  error,
  reset,
}: {
  error: Error & { digest?: string }
  reset: () => void
}) {
  return (
    <div>
      <h2>Something went wrong!</h2>
      <button onClick={() => reset()}>Try again</button>
    </div>
  )
}
```